safepay-typescript-client/src/apis/OrderApi.ts

```
@@ -0,0 +1,244 @@
+    /**
+     * <p>Perform the <code>REVERSAL</code> action. The request body may be empty. Note that this action is expected for the regular 3DS flow where the customer is redirected to your checkout after enrollment.</p> <p>This is relevant for the <code>instrument</code> mode used to save a payment method. In this scenario, the authoirzation hold on the customer\'s funds must be removed by requesting a reversal.</p> <p>This request must be authenticated by a Safepay shopper or merchant shopper.</p> 
+     * Reversal
+     */
+    async postReversalRaw(requestParameters: PostReversalRequest, initOverrides?: RequestInit | runtime.InitOverrideFunction): Promise<runtime.ApiResponse<object>> {
+        const queryParameters: any = {};
+
+        const headerParameters: runtime.HTTPHeaders = {};
+
+        headerParameters['Content-Type'] = 'application/json';
+
+        if (this.configuration && this.configuration.accessToken) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)